android/app/src/main/res/layout/content_session_detail.xml

```diff
@@ -0,0 +1,203 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.v4.widget.NestedScrollView
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    app:layout_behavior="@string/appbar_scrolling_view_behavior"
+    tools:showIn="@layout/activity_sessions_detail"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:id="@+id/nested_scrollview_session_detail"
+    tools:context=".activities.SessionDetailActivity">
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:orientation="vertical"
+        android:paddingLeft="@dimen/content_detail_margin">
```

Comment:
So this was inspired by Google I/O official app which is open source, take a look at the screenshot here: https://s19.postimg.org/h6rl91pdv/Screenshot_from_2016_12_25_17_30_27.png

That padding(64dp) is the whitespace provided below the FAB. I'm planning to overhaul the speaker recycler view part for the implementation similar design in next PR.